.github/workflows/macos-all.yml

```diff
@@ -1,10 +1,24 @@
 name: macOS-{10.15-11.0}
 
-on: [push, pull_request]
```

Comment:
Very neat, that means, CI is run every time you push to master and every time you push to a PR that is against master, correct?

Reply:
I think it's every push to master and every push to a pull request (or when you open a PR it will also start building that last commit to that branch)